### PR TITLE
feat(o-shop): add product form, modify form property values

### PR DESCRIPTION
**Pull request by jesus-mdz**
Add ProductForm component, adjusted exisiting Form property values. Minor adjustments with imports.

### CODE REVIEW

File: src/components/Auth/ProductForm.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      event.preventDefault();
+    }
+  };
+
+  return (
+    <div className="d-flex">
+      <Form
+        type="text"
+        title="Add New Product"
+        inputArray={[
```
Comment (ahaly92) [inline src/components/Auth/ProductForm.js]: pull this array into a const file in the Auth folder and import it in here

File: src/components/Auth/RegisterForm.js [hunk 2]
```diff
@@ -26,7 +26,33 @@ function RegisterForm() {
     <Form
       type="text"
       title="Register"
-      inputArray={['firstName', 'lastName', 'username', 'password']}
+      inputArray={[
```
Comment (ahaly92) [inline src/components/Auth/RegisterForm.js]: same here

File: src/components/Auth/ProductForm.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          },
+          {
+            title: 'Image Url',
+            name: 'imageUrl',
+            type: 'input',
+            placeholder: '',
+          },
+        ]}
+        primaryButton={{ title: 'Save Product' }}
+        restart={() => {
```
Comment (ahaly92) [inline src/components/Auth/ProductForm.js:36]: can you make this an optional parameter so you don't have to pass in a function that doesnt do anything

**Review by ahaly92 — CHANGES_REQUESTED** [review]
Nice work! A lot of really great stuff here. Left a few comments

**Review by ahaly92 — APPROVED** [review]

**Review by mmevans — APPROVED** [review]

**Pull request merged**